queries.sql

```diff
@@ -2,9 +2,9 @@
 
 SELECT * from animals WHERE name LIKE '%mon';
 SELECT * from animals WHERE TO_CHAR(date_of_birth, 'yyyy') BETWEEN '2016' and '2019';
-SELECT name from animals where neutered = false and escape_attempts > 3;
+SELECT name from animals where neutered = false and escape_attempts < 3;
```

Comment:
- [OPTIONAL] I appreciate that you implemented the changes 💯, However,  the neutered here should be true `neutered = true`. But for the hard work you've done, your project deserves to be approved, and I am sure that you will fix this minor issue. 👍

Reply:
Thanks for pointing that out again. Cheers